### PR TITLE
Fix bug where resp_load wouldn't load the response.kef file into ph5

### PR DESCRIPTION
**Pull request by nick-iris**
### What does this PR do?
Addresses #353. The `resp_load` command now successfully loads RESP files and updates the response_t table. The program still isn't that great, but works better than it did nonetheless.

### Checklist
- [x] This PR is not directly related to an existing issue (which has no PR yet).
- [x] All tests pass.
- [x] Any new or changed features have are documented.
- [x] Changes have been added to `CHANGELOG.txt` .
- [x] First time contributors have added your name to `CONTRIBUTORS.txt` .

### CODE REVIEW

File: ph5/utilities/resp_load.py [hunk 1]
```diff
@@ -138,11 +144,6 @@ def create_list(self):
                             bit_weight = Response_t['bit_weight/value_d']
                             bit_weight_units = Response_t['bit_weight/units_s']
                             gain_units = Response_t['gain/units_s']
-                        else:
-                            LOGGER.warning(
-                                "No Response table found for das {0} channel "
-                                "{1}.\n".format(serial, channel))
-                        try:
```
Comment (dsentinel) [inline ph5/utilities/resp_load.py]: Why remove the warning?

File: ph5/utilities/resp_load.py [hunk 2]
```diff
@@ -159,9 +160,30 @@ def create_list(self):
...
+                                    '',
+                                    '',
+                                    serial))
+                            LOGGER.debug(
+                                "No Response table found for das {0} channel "
+                                "{1}.".format(serial, channel))
                             continue
+        if not stations:
+            LOGGER.warning("No stations need updating. Is this the first time "
+                           "loading a response for the selected arrays?")
```
Comment (dsentinel) [inline ph5/utilities/resp_load.py]: Just curious; why ask if first time?

**Comment by dsentinel** [comment]
Does this address the problems described in #334?
Or do you have any insight?

**Comment by nick-iris** [comment]
I don't think I addressed any of the issues in #334. The program would flat out fail to run before these changes. It could still use some work if @damhuonglan has time to look at it.

File: ph5/utilities/resp_load.py (hunk 1 above)
Comment (hrotman-pic) [inline ph5/utilities/resp_load.py]: Please keep the warning.  It assists with troubleshooting.

Comment (nick-iris) [reply ph5/utilities/resp_load.py]: If you run the program on a master.ph5 file that hasn't had data loaded yet you'll potentially receiver thousands of lines of warnings. IMO, this just becomes noise.

Is there a reason that data needs to be loaded before a response is added to the master.ph5 file? Why should it make a difference if someone loads metadata into the master.ph5 before loading the timeseries?

Comment (nick-iris) [reply ph5/utilities/resp_load.py]: If you run the `resp_load` program on a master.ph5 file that hasn't had time series data loaded yet, you'll potentially receiver thousands of lines of warnings. IMO, this just becomes noise. The `ph5_validate` program will tell you if there is a das without data, but I don't think this is the job of `resp_load`.

Is there a reason that data needs to be loaded before a RESP file is added to a master.ph5? Does it matter if someone loads metadata into a master.ph5 before loading the time series?

Comment (hrotman-pic) [reply ph5/utilities/resp_load.py]: The PIC-distributed procedure tells users that data loading is one of the very first steps.  So it is current SOP to expect that data will be there when we run resp_load on a normal experiment.

Sure, ph5_validate can catch it, but by that time I've got more kefs I have to go back and check and probably delete and generate again.  It is easier to catch it at resp_load, and ph5_validate has enough buggy warnings (like the 'no data for this channel') that a warning about a das having no response table could become lost in the noise.

The question is now what harm is caused by keeping the warning as it is in resp_load until other bugs (like response_table_n_i indexing and the ph5_validate one I mentioned above) are smoothed out.  I would suggest that it causes no harm and doesn't impact the usefulness of resp_load for an archive being built according to current SOP.

Comment (nick-iris) [reply ph5/utilities/resp_load.py]: Thanks Holly. Would you mind if I just add a flag where a user can optionally silent warnings?

**Comment by nick-iris** [comment]
Fixed by #373

**Pull request closed without merging**